File: ServerFront/src/comm/QuoteRequest.java
package comm;

import java.io.Serializable;

public class QuoteRequest implements Serializable {
	private static final long serialVersionUID = -9044538308453434301L;

	private String[] instruments = new String[0];

	public String[] getInstruments() {
		return instruments;
	}

	public void setInstruments(String[] instruments) {
		this.instruments = instruments;
	}
}

File: iPhone/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/json/QuoteRequest.m
//
//  QuoteRequest.m
//  JEDIv7-CSTSv3-ClientAPI
//
//  Created by baolin on 13/5/24.
//  Copyright (c) 2013年 wangyubo. All rights reserved.
//

#import "QuoteRequest.h"

NSString * quoteRequest_jsonId          =@"QuoteRequest";
NSString * quoteRequeset_instruments    =@"1";

@implementation QuoteRequest

- (id)init
{
    self = [super init];
    if (self) {
        [super setEntry:@"jsonId" entry:quoteRequest_jsonId];
    }
    return self;
}

-(NSMutableArray *)getInstruments{
    @try {
        return [super getEntryArray:quoteRequeset_instruments];
    }
    @catch (NSException *exception) {
        return nil;
    }
}

-(void)setInstruments:(NSMutableArray *)instruments
{
    [super setEntry:quoteRequeset_instruments entry:instruments];
}

@end
